java/cloud/unum/usearch/Index.java

```diff
@@ -296,7 +307,11 @@ public Config expansion_search(long _expansion_search) {
       System.loadLibrary("usearch"); // used for tests. This library in classpath only
     } catch (UnsatisfiedLinkError e) {
       try {
-        NativeUtils.loadLibraryFromJar("/usearch/libusearch.so");
+        if (System.getProperty("os.name").equals("Mac OS X")) {
```

Comment:
This and the gradle changes were made to make it easier to develop on mac os. I've put these in a separate commit so it should be easy to drop them if desired.